phpBB/styles/prosilver/template/bbcode.html

```diff
@@ -11,6 +11,39 @@
 <!-- BEGIN quote_username_open --><blockquote><div><cite>{USERNAME} {L_WROTE}{L_COLON}</cite><!-- END quote_username_open -->
 <!-- BEGIN quote_open --><blockquote class="uncited"><div><!-- END quote_open -->
 <!-- BEGIN quote_close --></div></blockquote><!-- END quote_close -->
+<!-- BEGIN quote_extended -->
+<blockquote>
+	<xsl:if test="not(@author)">
```

Comment:
Ok, you're not talking about the template are you? The BBCode accepts any number of parameters but you won't have the "X wrote:" line if there's no author provided.